content/_data/changelogs/weekly.yml

```diff
@@ -8480,6 +8480,37 @@
   # pull: 4846 (PR title: tests: minor refactorings either for performance or readability)
   # pull: 4856 (PR title: Add .factorypath to .gitignore)
 
+- version: 2.249
+  date: 2020-07-24
+  changes:
+  - type: major bug
+    category: bug
+    pull: 4864
+    issue: 63168
+    authors:
+    - daniel-beck
+    message: |-
+      Do not throw exceptions when building environment for certain build steps (regression in 2.248).
+      In particular, the Powershell step from the Powershell plugin was affected.
```

Comment:
First ever entry in which the "(regression in …)" is not at the end of the text.

Also, we've never investigated whether other similarly popular plugins like NodeJS aren't affected.